### PR TITLE
fixes #1

### LINKED ISSUE #1: Add option to use -noaccurate_seek which dramatically speeds things up
-noaccurate_seek option uses nearest keyframe and is about a brazillian times faster.

### PR DESCRIPTION
**Pull request by mattbodman**
Added option to use faster keyframe approach to seeking

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,24 +9,38 @@ module.exports = (opts) => {
...
     ]
 
-    cmd
-      .outputOptions(outputOptions)
-      .output(output)
-      .on('start', (cmd) => log && log({ cmd }))
-      .on('end', () => resolve())
-      .on('error', (err) => reject(err))
-      .run()
+    cmd.
```
Comment (transitive-bullshit) [inline index.js]: please revert this change

**Comment by transitive-bullshit** [comment]
This looks great @mattbodman -- definitely a very welcome improvement! 😄 

Just one small note not to introduce unrelated style changes into PRs. Reverting that change should also fix the CI but you can run `npm run test` locally to make sure it's resolved.

Thanks!!

File: index.js (hunk 1 above)
Comment (mattbodman) [reply index.js]: I blame my IDE!  Sorry about that.  Cheers

**Comment by mattbodman** [comment]
@transitive-bullshit any chance of a merge?  Would love to have this change in the package.

**Comment by transitive-bullshit** [comment]
Hey sorry for the delay. This is an excellent addition.

Just released as `v1.0.2`.

Thanks again!
